include/matx/core/tensor_utils.h

```diff
@@ -440,6 +440,40 @@ namespace detail {
     }
   }
 
+  /**
+   * convert Type to string
+   *
+   * function convert a tensor type to a string
+   *
+   */
+  template <typename T> static std::string GetTensorType()
+  {
+    if constexpr (std::is_same_v<T, int32_t>)
```

Comment:
I would change these to be the real C++ datatypes. The ones from pybind.h were specific to numpy, but it's useful in c++ to see things like int32_t instead of i4.

Reply:
ok I will update. I didn't know if this was shorthand people preferred that just happened to line up with python types. I'll make them the real types